bin/fby_client

```diff
@@ -40,57 +24,60 @@ def get_sys_info():
 
 class FbyRunner(object):
 
-    def __init__(self, var_path=None):
+    def __init__(self, root=None, config_file=None, var_path=None):
+        if not root:
```

Comment:
If you understand what this does, maybe explain it in a comment?

    self.root = os.path.abspath(os.path.join(os.path.dirname(__file__), "../"))

Is particularly cryptic. Not because it's hard to read, but because it's hard to understand why root is the parent folder.